rd-net/Lifetimes/Serialization/UnsafeReader.cs

```diff
@@ -53,12 +53,10 @@ public UnsafeReader Reset(byte* ptr, int len)
 
     public void Skip(int bytes)
     {
-      AssertLength(bytes);
+      if (Mode.Assertion) AssertLength(bytes);
```

Comment:
yes, now it's possible, thanks. But that wasn't possible when the body of AssertLenght was huge, before this PR. And it doesn't work without AggressiveInlining thought.
I've checked the benchmarks and will make a new PR soon since this one is already merged.